frontend/src/components/eMIB/EditActionDialog.jsx

```diff
@@ -108,19 +113,127 @@ class EditActionDialog extends Component {
     this.setState({ action: updatedAction });
   };
 
-  showCancelConfirmationDialog = () => {
-    this.setState({ showCancelConfirmationDialog: true });
+  handleCancelConfirmationDisplay = () => {
```

Comment:
I think this function needs a couple sentences in English explaining the expected behavior.

Reply:
Added comments in commit #https://github.com/code-for-canada/project-thundercat/pull/229/commits/2ffba3713cadbb590f4fcc512a45eaaa15c24c65